docs/sources/enterprise/configuration.md

```diff
@@ -0,0 +1,187 @@
+
+Set to complete url to override menu logo.
+
+### fav_icon
+
+Set to complete url to override fav icon (icon shown in browser tab).
+
+### apple_touch_icon
+
+Set to complete url to override apple/ios icon.
+
+### footer_links
+
+List the links id's to use here, Grafana will look for matching links configurations the link id's should be space separated and contain no whitespace.
+
+## [meta_analytics]
+
+### max_file_age
+
+Max age for data files before they get deleted.
+
+### max_data_directory_size
+
+Max size in megabytes of the data files directory before files gets deleted.
+
+### data_path
+
+The directory where events will be stored in.
+
+## [analytics.summaries]
+
+### buffer_write_interval
+
+Interval for writing dashboard usage stats buffer to database.
+
+### buffer_write_timeout
+
+Timeout for writing dashboard usage stats buffer to database.
+
+### rollup_interval
+
+Interval for trying to rollup per dashboard usage summary only rolled up at most once per day.
+
+### rollup_timeout
+
+Timeout for trying to rollup per dashboard usage summary.
+
+## [analytics.views]
+
+### recent_users_age
+Age for recent active users.
+
+## [reporting]
+
+### rendering_timeout
+
+Timeout for each panel rendering request.
+
+### concurrent_render_limit
+
+Maximum number of concurrent calls to the rendering service.
+
+### image_scale_factor
+
+Scale factor for rendering images. Value `2` is enough for monitor resolutions, `4` would be better for printed material. Setting a higher value affects performance and memory.
+
+## [auth.saml]
+
+### enabled
+
+If true, the feature is enabled. Defaults to false.
+
+### certificate
+
+Base64-encoded public X.509 certificate. Used to sign requests to the IdP.
+
+### certificate_path
+
+Path to the public X.509 certificate. Used to sign requests to the IdP.
+
+### private_key
+
+Base64-encoded private key. Used to decrypt assertions from the IdP.
+
+### private_key_path
+
+Path to the private key. Used to decrypt assertions from the IdP.
+
+### idp_metadata
+
+Base64-encoded IdP SAML metadata XML. Used to verify and obtain binding locations from the IdP.
+
+### idp_metadata_path
+
+Path to the SAML metadata XML. Used to verify and obtain binding locations from the IdP.
+
+### idp_metadata_url
+
+URL to fetch SAML IdP metadata. Used to verify and obtain binding locations from the IdP.
+
+### max_issue_delay
+
+Duration, since the IdP issued a response and the SP is allowed to process it. Defaults to 90 seconds.
+
+### metadata_valid_duration
+
+Duration, for how long the SP's metadata should be valid. Defaults to 48 hours.
+
+### assertion_attribute_name
+
+Friendly name or name of the attribute within the SAML assertion to use as the user's name.
+
+### assertion_attribute_login
+
+Friendly name or name of the attribute within the SAML assertion to use as the user's login handle.
+
+### assertion_attribute_email
+
+Friendly name or name of the attribute within the SAML assertion to use as the user's email.
```

Comment:
```suggestion
Friendly name or name of the attribute within the SAML assertion to use as the user email.
```

Reply:
You get the idea. Please make "user" non-possessive, since we are referring, at least inderectly, to "user <field>" for all of these mappings.